.github/workflows/TestAcquia.yml

```diff
@@ -0,0 +1,90 @@
+name: "Test Acquia"
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+concurrency: development
+
+jobs:
+  Test-Acquia:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Create a Drupal project
+        run: composer create-project drupal/recommended-project . --ignore-platform-req=ext-gd
+
+      - uses: actions/checkout@v4
+        with:
+          path: drainpipe
+
+      - uses: ./drainpipe/scaffold/github/actions/common/set-env
+
+      - name: Install DDEV
+        uses: ./drainpipe/scaffold/github/actions/common/ddev
+        with:
+          git-name: Drainpipe Bot
+          git-email: no-reply@example.com
+          ssh-private-key: ${{ secrets.ACQUIA_SSH_PRIVATE_KEY }}
+
+      - name: Setup Project
+        run: |
+          ddev config --auto
+          ddev start
+          ddev composer config extra.drupal-scaffold.gitignore true
+          ddev composer config --json extra.drupal-scaffold.allowed-packages \[\"lullabot/drainpipe\"]
+          ddev composer config --no-plugins allow-plugins.composer/installers true
+          ddev composer config --no-plugins allow-plugins.drupal/core-composer-scaffold true
+          ddev composer config --no-plugins allow-plugins.lullabot/drainpipe true
+          ddev composer config repositories.drainpipe --json '{"type": "path", "url": "drainpipe", "options": {"symlink": false}}'
+          ddev composer config extra.drainpipe.acquia --json '{"settings": true, "github": []}'
+          ddev composer config minimum-stability dev
+          ddev composer require lullabot/drainpipe --with-all-dependencies
+
```

Comment:
@beto-aveiga it's a ddev bug, will be fixed in the next release. We fixed it on main for https://github.com/Lullabot/drainpipe/pull/729 and you can do something similar here.

Reply:
@beto-aveiga during the Drainpipe sync today it was mentioned that our workaround for DDEV is no longer needed because it's been addressed in the latest release ([v1.24.0](https://github.com/ddev/ddev/releases/tag/v1.24.0)). We can remove any workaround that we added (even though it would work fine if we left it in).